GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -74,21 +73,31 @@ public QueryController(GitHubClient gitHubClient)
                 State = ItemStateFilter.Open,
             };
 
+            if (!string.IsNullOrWhiteSpace(labels) && !(labels == "undefined")) {
+                string[] labelvalues = labels.Split(',');
+                foreach (var label in labelvalues)
+                {
+                    issueRequest.Labels.Add(label);
+                }
+            }
+
             var issues = await _gitHubClient.Issue.GetAllForRepository(owner, repository, issueRequest);
             issues = issues.Where(i => i.PullRequest == null).ToList();
             return issues;
         }
     }
 
+
     public class AssigneeCount
     {
-        public string assignee;
-        public int count;
+        public string Assignee;
```

Comment:
I suspect that these changes will break the chart functionality at something like https://githubdashboard-dev-as.azurewebsites.net/assignedchart/dotnet/project-system?milestone=15.7

Reply:
OK.  I'll roll it back.  I didn't like the warnings.